packages/eslint-plugin/src/rules/require-dispatched-action-is-called.ts

```diff
@@ -22,52 +16,47 @@ const rule: TSRuleModule = {
   },
 
   create(context: TSRuleContext) {
-    let importsDispatchFromModel: boolean = false;
-    let dispatchImportSymbol: ts.Symbol;
-    const parserServices: ParserServices = tsPluginUtil.getParserServices(
-      context,
-    );
-
-    const checker = parserServices.program!.getTypeChecker();
     return {
-      ImportDeclaration: (node: TSESTree.ImportDeclaration): void => {
-        const specifiers = node.specifiers;
-        if (node.source.type === AST_NODE_TYPES.Literal) {
-          if (matchModel(node.source.value as string)) {
-            specifiers.forEach(specifier => {
-              if (specifier.type === AST_NODE_TYPES.ImportSpecifier) {
-                const specifierTsNode = parserServices.esTreeNodeToTSNodeMap!.get<
-                  TSNode
-                >(specifier);
-                if (specifierTsNode.getFullText() === "dispatch") {
-                  importsDispatchFromModel = true;
-                  dispatchImportSymbol = (specifierTsNode as any).symbol;
-                }
-              }
-            });
-          }
-        }
-      },
-      CallExpression: (node: TSESTree.CallExpression): void => {
+      "CallExpression:not(CallExpression > CallExpression.callee)"(
+        node: TSESTree.Node,
+      ) {
         if (
-          importsDispatchFromModel &&
-          node &&
-          node.callee.type === "Identifier" &&
-          node.callee.name === "dispatch" &&
-          node.parent &&
-          node.parent.type !== "CallExpression"
+          node.type !== "CallExpression" ||
+          (node.type === "CallExpression" &&
```

Comment:
This line isn't necessary

Reply:
I need it for the types.  Returning if `node.type !== "CallExpression"` means I can access `node.callee` in the body. The `node.type === "CallExpression"` check is to access the callee in the if condition.